models/sessions/scratch/snowflake/snowplow_web_sessions_this_run.sql

```diff
@@ -181,6 +215,17 @@ select
     -- engagement fields
     c.page_views,
     c.engaged_time_in_s,
+    {%- if var('snowplow__list_event_counts', false) %}
+    TRY_PARSE_JSON(c.event_counts_string) as event_counts,
+    {%- endif %}
+    c.total_events,
+    c.page_views >= 2 or c.engaged_time_in_s / {{ var('snowplow__heartbeat', 5) }} >= 2
```

Comment:
default of 10 used elsewhere
```suggestion
    c.page_views >= 2 or c.engaged_time_in_s / {{ var('snowplow__heartbeat', 10) }} >= 2
```